protocol/0038-OLIQ-liquidity_provision_order_type.md

```diff
@@ -60,10 +60,20 @@ Input data:
 
 Steps:
 
+1. From the market parameter `market.liquidity.volumeRange` which is a percentage price move (e.g. `0.05 = 5%` and from `mid_price` calculate:
```

Comment:
Not sure where we specify, but I assume we want this parameter amendable with a market update (assuming it's a market parameter) right?

Reply:
Yes, I've updated the text.